doc/source/jbnmr_examples/s6_seperate_inner_phase.rst

```diff
@@ -5,6 +5,6 @@ The data used in this example is available for
 
 Listing S6
 
-[:download:`seperate.py <../../../examples/jbnmr_examples/s6_seperate_inner_phase/seperate.py>`]
+[:download:`separate.py <../../../examples/jbnmr_examples/s6_seperate_inner_phase/separate.py>`]
 
-.. literalinclude:: ../../../examples/jbnmr_examples/s6_seperate_inner_phase/seperate.py
+.. literalinclude:: ../../../examples/jbnmr_examples/s6_seperate_inner_phase/separate.py
```

Comment:
this will break the link

Reply:
I can't explain this. My local files do contain `seperate`. Despite `push -f`, the files on the remote repository are different, they contain `separate`. Will keep investigating.

Reply:
Fixed.